HelloMixpanel/HelloMixpanel.xcodeproj/project.pbxproj

```diff
@@ -1226,9 +1473,9 @@
 				CLANG_WARN_SUSPICIOUS_IMPLICIT_CONVERSION = YES;
 				CODE_SIGN_ENTITLEMENTS = HelloMixpanel/HelloMixpanel.entitlements;
 				CODE_SIGN_IDENTITY = "";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = E8FVX7QLET;
```

Comment:
this is what it did when i wanted to build HelloMixpanel to the real phone/watch. is it bad to commit it because it's open source?